.github/workflows/ci.yml

```diff
@@ -96,7 +96,7 @@ jobs:
 
   build-bot:
     runs-on: ubuntu-latest
-    timeout-minutes: 20
+    timeout-minutes: 45
```

Comment:
we could restore this back as we have the fast build now

Reply:
reverted back to 20 in latest commit